pkg/networkservice/common/authorize/client.go

```diff
@@ -27,19 +32,37 @@ import (
 	"github.com/networkservicemesh/sdk/pkg/networkservice/core/next"
 )
 
-type authorizeClient struct{}
+type authorizeClient struct {
+	provider security.Provider
+}
+
+const (
+	defaultTokenExpiration = 5 * time.Minute
```

Comment:
As I remember we discussed something like 15 minutes, should we make this timeout configurable somehow?

Reply:
Sure, fixed